theseus/extlib/baspacho_solver.h

```diff
@@ -57,7 +79,7 @@ class NumericDecomposition {
                       const torch::Tensor& inds);
 
     // apply damping, ie on factor diag x -> x*(1+alpha) + beta (Cuda)
-    void damp_cuda(double alpha, double beta);
+    void damp_cuda(double* alpha, double* beta);
```

Comment:
Hmm alpha/beta here are double while elsewhere are torch::Tensor. Should we add something to the name to reduce ambiguity: alpha_double or alpha_tensor or something? Is this a concern?

Reply:
This is the CUDA kernel for damping, so I think this is fine. It should be the only place where this happens.